src/index.html

```diff
@@ -11,6 +11,36 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form action="#" method="get"
+      data-qa="big">
```

Comment:
```suggestion
    <form
      action="#"
      method="get"
      data-qa="big"
    >
```
<img width="1038" alt="Screenshot 2022-12-01 at 11 18 16" src="https://user-images.githubusercontent.com/94289038/205014272-2e819966-1453-4e4b-b3ad-66f4c0b4ba31.png">